menus/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
Minor: Can this be moved to the other gitignore, just to keep them all in same place? Or is it there cause we have to have the folder present?

Reply:
Yeah the folder has to be present, it didn't work having it in the other gitignore because git wasn't keeping the empty folder.

Reply:
Yep that makes sense then! Did you try 'menu/*' ?  Think that will just ignore contents